src/map/handler/Map.TouchZoom.js

```diff
@@ -109,6 +109,15 @@ L.Map.TouchZoom = L.Handler.extend({
 		    .off(document, 'touchmove', this._onTouchMove)
 		    .off(document, 'touchend', this._onTouchEnd);
 
+		var snap = L.Browser.any3d ? this._map.options.zoomSnap : 1;
+		if (snap) {
```

Comment:
In my test above edge will return a value if 2 because it match (L.Browser.win && L.Browser.chrome). So I think they are tightly coupled (agree?).
What to do now? Remove my pull request completely before I follow your general rule description?